sklearn/_build_utils/tempita.py

```diff
@@ -0,0 +1,57 @@
+import argparse
```

Comment:
Did we take this file from another repository?

Reply:
I see this is also in NumPy, we could have a header mentioning which version it was taken and the location in case we just want to make some update time to time.

Reply:
It is indeed taken from a repo (maybe scipy?) and adapted slightly. For example our template files ends with `.tp` and other projects seem to use `.in`. IIRC I also had some encoding issues when I tried testing on Windows with `tempita.Template.from_filename`.

I would say this is unlikely to change much, this is mostly doing creating a `tempita.Template` from a file and calling `.substitute` on it.